styling-and-themes/overview.md

```diff
@@ -13,82 +13,80 @@ Telerik UI for .NET MAUI comes with a built-in theme that controls the visual ap
 
 ## Theme
 
-A *theme* is a collection of styles and templates in XAML files, which determine the appearance of the Telerik .NET MAUI components, including fonts, colors, sizes and layouts.
+A *theme* is a collection of styles and templates in XAML files, which determine the appearance of the Telerik .NET MAUI components, including fonts, colors, sizes, and layouts.
 
 ## Swatch
 
 A *theme swatch* is a color variation of a theme. All swatches of a given theme use the same fonts, sizes, and layouts. On the other hand, the text colors, background colors and border colors are different. For example, *Purple* and *Purple Dark* are two built-in swatch names.
 
-The Telerik .NET MAUI theme comes with a set of 8 predefined swatches for both dark and light modes: `Main`, `Main Dark`, `Ocean Blue`, `Ocean Blue Dark`, `Purple`, `Purple Dark`, `Turquoise` and `Turquoise Dark`.
+The Telerik .NET MAUI theme comes with a set of eight predefined swatches for both dark and light modes: `Main`, `Main Dark`, `Ocean Blue`, `Ocean Blue Dark`, `Purple`, `Purple Dark`, `Turquoise` and `Turquoise Dark`.
 
 Here is an example of the AutoComplete control with `Purple` and `Purple Dark` applied:
 
 ![Telerik .NET MAUI Theming](images/theming-default.png)
 
-### Comparing Swatches
+>You can explore and compare the built-in theme swatches in the [Telerik .NET MAUI ControlsSamples App]({%slug controls-samples-app%}). Go to the Theming example of each component and use the **Change Theme** dropdown to switch between the theme swatches.
 
-You can explore and compare the built-in theme swatches in the [Telerik .NET MAUI ControlsSamples App]({%slug controls-samples-app%}). Go to the Theming example of each component and use the "Change Theme" dropdown to switch between the theme swatches.
-
-## Using the Telerik Theming
+## Using the MAUI Themes
 
 To enable the Telerik Theming in your app, follow the steps below (assuming you already have a .NET MAUI app set up to work with Telerik .NET MAUI controls):
 
-**1.** Go to the `.csproj` file of your MAUI project and set `UseTelerikTheming` property to `true` in a separate `PropertyGroup`:
+1. Go to the `.csproj` file of your MAUI project and set `UseTelerikTheming` property to `true` in a separate `PropertyGroup`:
 
-```XAML
-<PropertyGroup>
-	<UseTelerikTheming>true</UseTelerikTheming>
-</PropertyGroup>
-```
+    ```XAML
+    <PropertyGroup>
+        <UseTelerikTheming>true</UseTelerikTheming>
+    </PropertyGroup>
+    ```
 
-**2.** Rebuild the solution - this will generate a new **TelerikTheming** folder inside the project containing all the styles and resources needed for the Telerik .NET MAUI controls:
+1. Rebuild the solution - this will generate a new `TelerikTheming` folder inside the project containing all the styles and resources needed for the Telerik .NET MAUI controls:
```

Comment:
```suggestion
1. Rebuild the solution&mdash;this will generate a new `TelerikTheming` folder inside the project containing all the styles and resources needed for the Telerik .NET MAUI controls:
```